sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/pipeline/ByteBufferOutputStream.java

```diff
@@ -8,15 +8,15 @@
 import java.util.ArrayList;
 import java.util.List;
 
-class ByteBufferOutputStream extends OutputStream {
+public class ByteBufferOutputStream extends OutputStream {
 
     private final AppInsightsByteBufferPool byteBufferPool;
 
     private final List<ByteBuffer> byteBuffers = new ArrayList<>();
 
     private ByteBuffer current;
 
-    ByteBufferOutputStream(AppInsightsByteBufferPool byteBufferPool) {
+    public ByteBufferOutputStream(AppInsightsByteBufferPool byteBufferPool) {
```

Comment:
are these changes needed?

Reply:
reverted.. earlier change requires this to be public.